pubspec.yaml

```diff
@@ -3,7 +3,7 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 0.1.5
+version: 0.2.0
```

Comment:
I think this should be something like `0.2.0-dev` for commits that are not `0.2.0`.

https://semver.org/spec/v2.0.0-rc.1.html#spec-item-10

Reply:
Updated.